docs/4-dca/2-dca-sdk.md

```diff
@@ -52,8 +52,8 @@ async function createDCA() {
     cycleSecondsApart: BigInt(86400), // 1 day between each order -> 60 * 60 * 24
     inputMint: USDC, // sell
     outputMint: BONK, // buy
-    minOutAmountPerCycle: null,  // effectively allows for a min price. refer to Integration doc
-    maxOutAmountPerCycle: null, // effectively allows for a max price. refer to Integration doc
+    minOutAmountPerCycle: null,  // effectively allows for a max price. refer to Integration doc
```

Comment:
Can you give an explanation of why the comments are changed?

Reply:
@cvgit-p yes!
Setting `minOutAmountPerCycle` actually sets a max price and setting `maxOutAmountPerCycle` actually sets a min price.
For example:
User wants to buy sol at max price $20 USDC and the inAmountPerCycle is $5
minOutAmountPerCycle = inAmountPerCycle / maxPrice = 5 / 20 = 0.75.
The user will get a minimum amount of 0.75 Sol allows for a max price of $20
I think this aligns with how it looks in the UI: setting minOutAmountPerCycle is done by setting max price in the UI
<img width="761" alt="image" src="https://github.com/jup-ag/space-station/assets/4204671/f7814789-d4bb-45b2-ae33-f0d0f0de7c58">